website/frontend/src/pages/CareerPage.js

```diff
@@ -9,151 +9,161 @@ import { isEmpty } from 'underscore';
+                      {department.name}
+                    </span>
+                  ))
                 ) : (
-                    <div className="content">
-                        <div className="container">
-                            <div className="label">Categories</div>
-                            <div className="tags">
-                                <span className={selectedTagClassName('all')} onClick={onTagClick('all')}>
-                                    Open positions
-                                </span>
-                                {departments.length > 0 ? (
-                                    departments.map((department) => (
-                                        <span
-                                            className={selectedTagClassName(department.name)}
-                                            key={department.id}
-                                            onClick={onTagClick(department.name)}>
-                                            {department.name}
-                                        </span>
-                                    ))
-                                ) : (
-                                    <span></span>
-                                )}
-                            </div>
-                            {groupedKeys.length > 0 ? (
-                                groupedKeys.map((groupedKey) => {
-                                    const departmentListing = groupedListing[groupedKey];
-                                    return (
-                                        <DepartmentListing
-                                            department={groupedKey}
-                                            listing={departmentListing}
-                                            key={groupedKey}
-                                        />
-                                    );
-                                })
-                            ) : (
-                                <div className="no-listing">We currently have no open positions.    </div>
-                            )}
-
-                            <div className="self-intro">
-                                <span>
-                                    Don’t see a <br />
-                                    position that fits you perfectly? Introduce yourself here{' '}
-                                </span>
-                                <a href="mailto:careers@airqo.net" className='mail-link'>careers@airqo.net</a>
-                            </div>
-                        </div>
-                    </div>
+                  <span></span>
                 )}
+              </div>
+              {groupedKeys.length > 0 ? (
+                groupedKeys.map((groupedKey) => {
+                  const departmentListing = groupedListing[groupedKey];
+                  const listing = departmentListing.filter(
+                    (job) => new Date(job.closing_date).getTime() >= new Date().getTime()
```

Comment:
Filter job listings by newer dates in order to show on the frontend

Reply:
Great!